services/graphite/config.go

```diff
@@ -38,6 +38,7 @@ type Config struct {
 	BindAddress      string        `toml:"bind-address"`
 	Database         string        `toml:"database"`
 	Enabled          bool          `toml:"enabled"`
+	WriteSkip        bool          `toml:"write-skip"`
```

Comment:
Hmmm, hadn't though of the sample config. OK, let me nuke it, and deal with it another way (perhaps the environment variable).